config/v3_2_experimental/types/schema.go

```diff
@@ -2,6 +2,12 @@ package types
 
 // generated by "schematyper --package=types config/v3_2_experimental/schema/ignition.json -o config/v3_2_experimental/types/schema.go --root-type=Config" -- DO NOT EDIT
 
+type Clevis struct {
+	Tang      []Tang `json:"tang,omitempty"`
```

Comment:
Let's not block on this, but I think `fetch-offline` should probably also check whether any Tang pins are present and `SignalNeedNet()` in that case. We can do that in a follow-up though!

Hmm, and then the rootmap code could also check if LUKS devices in the root block device tree are Tang-pinned and adds `rd.neednet=1` to the BLS for subsequent boots.

Reply:
Whoops, yeah, forgot about `fetch-offline`.